### DISCUSSION TITLE
Are there any `DrawingArea` examples?

### DISCUSSION THREAD

**Discussion by Barugon** [gtk-rs/gtk4-rs#1398] [Q&A] [answered]
Title pretty much says it all. I tried researching how to draw using Cairo but it's very esoteric.

**Comment by sdroege** [comment, marked as answer]
https://www.cairographics.org has extensive documentation how to use cairo. It's a standard 2D graphics API, very similar to Skia and more or less everything else.

There are some examples in the gtk3-rs repository.

**Reply by Barugon** [reply]
Thanks.
